src/components/Article/Editor.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
```

Comment:
what purpose does this serve? i have seen you have another file named `ArticleEditor`